install-script-dependencies.sh

```diff
@@ -6,6 +6,11 @@ while getopts 'c:' flag; do
 	esac
 done
 
+if [ ! "$(realpath "$0")" ]; then
```

Comment:
`command -v` might have been a better conditional, but it looks like the CircleCI container doesn't have `command`

Reply:
Awesome!